rclrs_tests/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<?xml-model
+   href="http://download.ros.org/schema/package_format3.xsd"
+   schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>rclrs_tests</name>
+  <version>0.1.0</version>
+  <description>Package containing tests for rclrs that make use of msgs and other dependencies</description>
+  <maintainer email="christopherjreid@outlook.com">Chris Reid</maintainer>
+  <license>Apache License 2.0</license>
+
+  <build_depend>test_msgs</build_depend>
+  <build_depend>rclrs</build_depend>
+  <build_depend>rosidl_runtime_rs</build_depend>
+
+  <exec_depend>test_msgs</exec_depend>
```

Comment:
These dependencies should be declared as `<build_depend>` as well since we need them at build time. You can simplify this by just using `<depend>` which combines both build and exec dependencies.